crypto/cms/cms_sd.c

```diff
@@ -760,6 +766,10 @@ static int cms_SignerInfo_content_sign(CMS_ContentInfo *cms,
         unsigned char *sig;
         unsigned int siglen;
 
+        if (md != NULL) {
+            ERR_raise(ERR_LIB_CMS, CMS_R_CMS_R_OPERATION_UNSUPPORTED);
```

Comment:
You've duplicated `CMS_R_` here. Please fix that. Unfortunately I am afraid make update won't drop an existing reason code so you'd need to revert all the changes from the previous make update run that added this incorrect reason code first before running make update again.

Reply:
Another option is to manually fix the generated code in `cms_err.c`, `../err/oenssl.txt`, and `../../include/openssl/cmserr.h`.